cmd/main.go

```diff
@@ -26,21 +27,23 @@ type Handler interface { // nolint: deadcode
 
 func Run() {
 	logrus.SetFormatter(&logrus.JSONFormatter{})
-	mux := http.NewServeMux()
+	r := mux.NewRouter()
 	myData := repository.NewDataBase()
 	myRepo := repository.NewRepo(myData)
 	myServ := service.NewServ(myRepo)
 	myH := handler.NewHandler(myServ)
-	mux.Handle("/positions", myH)
-	mux.Handle("/position/", myH)
-	mux.Handle("/position", myH)
-	mux.Handle("/employees", myH)
-	mux.Handle("/employee/", myH)
-	mux.Handle("/employee", myH)
-	idMiddleware := middleware.IDMiddleware(mux)
-	timerMiddleware := middleware.TimeLogMiddleware(idMiddleware)
-	accessLogMiddleware := middleware.AccessLogMiddleware(timerMiddleware)
-	err := http.ListenAndServe("localhost:8080", accessLogMiddleware)
+	r.HandleFunc("/positions", myH.GetPositions).Queries("limit", "{limit:\\S+}", "offset", "{offset:\\S+}")
+	r.HandleFunc("/employees", myH.GetEmployees).Queries("limit", "{limit:\\S+}", "offset", "{offset:\\S+}")
+	r.HandleFunc("/position/{id:\\S+}", myH.GetPosition).Methods("GET")
```

Comment:
Why are you using `\\S+` in all patterns ?

Reply:
Since it is convenient and plain. The user will enter an id in these requests, which is 13 characters long. And it's easier to do it through this regex. Or I don't understand you?

Reply:
I can move this regex to a separate variable and add it to the path.

Reply:
`\\S+` doesn't perform the validation you are talking about. It only matches one or more whitespaces

Reply:
I know, that \\s+ - matches one or more whitespaces, but \\S+ mathces one or more characters except whitespaces.

Reply:
Yep, you are right. But ideally you anyway have to do validation in handlers of id to check if it a valid uuid